appear/src/main/java/com/arcbees/gquery/appear/Appear.gwt.xml

```diff
@@ -0,0 +1,3 @@
+<module>
+    <inherits name='com.google.gwt.query.Query'/>
```

Comment:
It is intentional because two reasons:
1.- Observe plugin could finally go to gQuery core, and I dont want to break code if this happens in the future.
2.- I'm considering that all plugins could be under the com/google/gwt/query/client/plugins package:
  -  so as simply adding the .jar dependency it works. 
  -  makes the gquery plugin system be more out-of-the-box like jQuery does.
  - It's not a mandatory practice, but recommended.
  - We could include easily any plugin in the core whenever we want if it goes popular